### PR TITLE
restartApp method + hotkeys test

### PR DESCRIPTION
**Pull request by holiber**

### CODE REVIEW

File: app/services/websocket.ts [hunk 1]
```diff
@@ -159,5 +159,7 @@ export class WebsocketService extends Service {
     if (this.appService.state.argv.includes('--network-logging')) {
       electronLog.log(`WS: ${message}`);
     }
+
+    electronLog.log('WEBSOCKET', message, args);
```
Comment (andycreeth) [inline app/services/websocket.ts]: Hmm is this needed?  Seems incongruous with the `--network-logging` switch above.

File: app/services/widgets/settings/widget-settings.ts [hunk 2]
```diff
@@ -157,7 +157,10 @@ export abstract class WidgetSettingsService<TWidgetData extends IWidgetData>
       body: req.body ? JSON.stringify(req.body) : void 0,
     });
 
-    return fetch(request).then(handleResponse);
+    electronLog.log('REQUEST TO', method, req.url, JSON.stringify(req.body));
+    const result = await fetch(request).then(handleResponse);
+    electronLog.log('RESPONSE FROM', method, req.url, result);
```
Comment (andycreeth) [inline app/services/widgets/settings/widget-settings.ts]: Needed?  Or are these just for debugging?

File: test/widgets/stream-boss.ts [hunk 3]
```diff
@@ -26,6 +26,7 @@ test('Set stream-boss health', async t => {
   t.pass();
 });
 
+// TODO: flaky
```
Comment (andycreeth) [inline test/widgets/stream-boss.ts]: Should this comment be down with the skipped test in this file?  I'm a little confused.

**Comment by holiber** [comment]
Ouch, some changes here are from the accidental merge with a wrong branch. Going to fix shortly

File: test/helpers/spectron/index.ts [hunk 4]
```diff
@@ -87,6 +87,14 @@ export interface ITestContext {
 
 export type TExecutionContext = ExecutionContext<ITestContext>;
 
+let startApp: (t: TExecutionContext) => Promise<any>;
+let stopApp: (clearCash?: boolean) => Promise<any>;
```
Comment (andycreeth) [inline test/helpers/spectron/index.ts]: `clearCache`?

**Pull request merged**